core/src/main/java/io/micronaut/core/execution/CompletableFutureExecutionFlow.java

```diff
@@ -38,8 +38,8 @@ public interface CompletableFutureExecutionFlow<T> extends ExecutionFlow<T> {
      * @return a new flow
      */
     @NonNull
-    static <K> ExecutionFlow<K> just(@NonNull CompletableFuture<K> value) {
```

Comment:
i hope this method isn't used outside core because this is a binary incompatible change. it's probably fine because this is Internal but it might be another servlet situation